bundles/InstallBundle/src/Command/InstallCommand.php

```diff
@@ -111,6 +111,11 @@ private function getOptions(): array
                 'default' => '',
                 'group' => 'db_credentials',
             ],
+            'install-bundles' => [
```

Comment:
Even when providing `--install-bundles` option, interactive prompt is still there to select bundles to install.